<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeEditorPropertyPage</class>
 <widget class="sf::PropertyPage" name="CodeEditorPropertyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeEditorPropertyPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbAppearance">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbShowGutter">
        <property name="toolTip">
         <string>Show a gutter with line numbering.</string>
        </property>
        <property name="text">
         <string>Show &amp;Gutter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cbDarkMode">
        <property name="toolTip">
         <string>Use darkmode colors in the editor.</string>
        </property>
        <property name="text">
         <string>&amp;Dark mode</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblFontType">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Font &amp;Type</string>
        </property>
        <property name="buddy">
         <cstring>cbFontType</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QFontComboBox" name="cbFontType">
        <property name="toolTip">
         <string>Select a monospaced font used by the editor.</string>
        </property>
        <property name="currentText">
         <string>Monospace</string>
        </property>
        <property name="fontFilters">
         <set>QFontComboBox::MonospacedFonts</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblFontSize">
        <property name="text">
         <string>&amp;Font &amp;Size</string>
        </property>
        <property name="buddy">
         <cstring>cbFontType</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="sbFontSize">
        <property name="toolTip">
         <string>Font size used by the editor.</string>
        </property>
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>36</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbUseHighLighting">
        <property name="toolTip">
         <string>Use highlighting in the editor.</string>
        </property>
        <property name="text">
         <string>Use &amp;Highlighting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sf::PropertyPage</class>
   <extends>QWidget</extends>
   <header>misc/qt/PropertyPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
